packages/react-core/src/components/Select/Select.tsx

```diff
@@ -548,10 +548,12 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
             hasClearButton={hasOnClear}
           >
             {customContent && (
-              <div className={css(styles.selectToggleWrapper)}>
-                {toggleIcon && <span className={css(styles.selectToggleIcon)}>{toggleIcon}</span>}
-                <span className={css(styles.selectToggleText)}>{placeholderText}</span>
-              </div>
+              <React.Fragment>
```

Comment:
Is the fragment necessary?

Reply:
No, it's not necessary. I will remove it, thanks.